infrastructure-as-code/aws-lambda/README.md

```diff
@@ -0,0 +1,8 @@
+# Terraforming EC2 lifecycles with AWS Lambda
+This Terraform configuration provides examples of:
+ - AWS Lambda functions for performing various ec2 cleanup tasks
```

Comment:
Will this guide ever expand beyond EC2 instances? Or is that all that's in scope for now?

Reply:
Yes, I've tried to design it in such a way that other resources will be easy to support.

Reply:
Awesome. So re: my previous comment, maybe make it `resource-cleanup-aws-lambda` or something more creative than that :)